CooperationHullMainSite/Views/Home/Index.cshtml

```diff
@@ -1,73 +1,108 @@
+                A planet in crisis. A failing economy. A divided society. Broken party politics.
+            </p>
+            <p>
+                This is the story of the world today and nobody (apart from the 1% at the top) 
+                benefits from keeping it this way. No ordinary person wants corrupt leaders, crop failures 
+                or the culture war and, to put it simply, there are more of us than there are of them. So…
+            </p>
+            <br />
+        </div>
+
+        <div class="col-lg-6 col-md-12 p-4">
+            <div class="feature-image">
+                <img src="~/assets/img/circles-diagram-transparent.png" alt="" />
+            </div>
+        </div>
+
+    </div>
+</section>
 
-            @if (Model.formSignedCounter >= 0)
-            {
-                <p>@($"{Model.formSignedCounter} have signed so far")</p>
-            }
-            else
-            {
-                <p>Hundreds of people have signed already</p>
-            }
 
 
-            <h1 class="heading-lg">THE BIG ANSWER</h1>
-            <p>People's Assemblies are a simple tool that will change everything.</p>
-            <p>Public meetings, happening everywhere, replacing party politics with people power.</p>
+<section class="homepage-block big-answer ">
+    <div class="row white">
+
+        <div class="col-lg-6 col-md-12 p-4">
+            <div class="feature-image">
+                <img src="~/assets/img/circles-diagram-transparent.png" alt="" />
+            </div>
+        </div>
+
+        <div class="col-lg-6 col-md-12 p-4">
+            <h1 class="heading-lg">The big answer</h1>
+            <p>
+                ...we propose a new story. In this version, the 99% works through our differences and chooses
+                to put all our wisdom, knowledge, love and technology to use. We choose to construct better systems,
+                ones that build us up rather than keep us down. 
+            </p>
             <br>
-            <p>The Hull People's Assembly will be the first of its kind in the UK.</p>
-            <p>And we need your help to pull it off.</p>
+            <p>
+                A healthy earth. A Solidarity Economy. Strong communities.
+                Local democracy.
+            </p>
             <br>
-            <p>We'll only launch when we have 2,000 signatures from local people.</p>
-            <p>Pledge your support to the Hull People's Assembly by signing here, now</p>
+            <p>
+                People’s Assemblies are a simple tool that will change everything. Public meetings,
+                happening everywhere, replacing party politics with people power.
+            </p>
             <br>
-            <a class="" asp-area="" asp-controller="Home" asp-action="Donate">Find out more about People's Assemblies'</a>
+            <a class="" asp-area="" asp-controller="Home" asp-action="TheBigIdea">Click here to find out more</a>
```

Comment:
The latest designs seem to have this link as inline with the last paragraph. I'm conscious you may want to change that after spending some time building out the page, but thought is was worth mentioning here anyway!

Reply:
Good spot, I've updated it